instrumentation/README.lto.md

```diff
@@ -88,16 +88,35 @@ apt-get install -y clang-12 clang-tools-12 libc++1-12 libc++-12-dev \
 ### Building llvm yourself (version 12)
 
 Building llvm from github takes quite some long time and is not painless:
-```
+```sh
 sudo apt install binutils-dev  # this is *essential*!
 git clone https://github.com/llvm/llvm-project
```

Comment:
my oversight - can you please add `--depth=1`? I dont want to push that and then your commit might conflict

Reply:
fixed in e9849855